src/java.desktop/share/classes/sun/swing/FilePane.java

```diff
@@ -1124,7 +1124,7 @@ public Component getTableCellEditorComponent(JTable table, Object value,
     class DetailsTableCellRenderer extends DefaultTableCellRenderer {
         JFileChooser chooser;
         DateFormat df;
-        final static double baseFileSize = 1000.0;
+        static final double baseFileSize = 1000.0;
         final MessageFormat mf = new MessageFormat("");
         final NumberFormat nf = NumberFormat.getNumberInstance();
```

Comment:
All these three new fields can be declared `private`, they aren't used anywhere else. For the sake of consistency, the existing fields `chooser` and `df` may also be declared `private` if they're unused outside of this class.

Reply:
@aivanov-jdk Updated the suggested changes.